tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -720,25 +730,18 @@ private static boolean shouldIgnoreUri(String uri) {
     return ImmutableMultimap.copyOf(builder);
   }
 
-  // Combine content of script tags into a group. To guarantee the correctness, it only groups
-  // content of `src`-less scripts between `src`-full scripts. The last combination gets inserted at the
-  // end of the document.
-  // e.g.,
-  //   <script>A</script>
-  //   <script>B</script>
-  //   <script src="srcful1"></script>
-  //   <script src="srcful2"></script>
-  //   <script>C</script>
-  //   <script>D</script>
-  //   <script src="srcful3"></script>
-  //   <script>E</script>
-  // gets compiled as
-  //   <script>A,B</script>
-  //   <script src="srcful1"></script>
-  //   <script src="srcful2"></script>
-  //   <script>C,D</script>
-  //   <script src="srcful3"></script>
-  //   <script>E</script>
+  /**
+   * Combine content of script tags into a group. To guarantee the correctness, it only groups
+   * content of `src`-less scripts between `src`-full scripts. The last combination gets inserted at
+   * the end of the document. e.g., <script>A</script> <script>B</script> <script
```

Comment:
Javadoc is interpreted as HTML, so these literal `<script>` tags will
get clobbered (sanitized away, hopefully). You can use `{@code ...}`
markup to write them safely:

```java
  /**
   * Combine content of script tags into a group. To guarantee the correctness, it only groups
   * content of `src`-less scripts between `src`-full scripts. The last combination gets inserted at
   * the end of the document. e.g., {@code <script>A</script> <script>B</script> <script
   * src="srcful1"></script> <script src="srcful2"></script> <script>C</script> <script>D</script>
   * <script src="srcful3"></script> <script>E</script>} is compiled as {@code <script>A,B</script>
   * <script src="srcful1"></script> <script src="srcful2"></script> <script>C,D</script> <script
   * src="srcful3"></script> <script>E</script>}
   *
```

You can also see that the lexical highlighting here and in your editor
makes it much easier to see where the `<script>` tags stop and start.